test/regression-evm/test-totalSupply.sh

```diff
@@ -5,13 +5,18 @@ exec kore-exec \
     --module VERIFICATION \
     --strategy all \
     --max-counterexamples 1 \
-    --smt-timeout 99 \
-    --smt-retry-limit 1 \
+    --smt-timeout 125 \
+    --smt-retry-limit 3 \
     --smt-reset-interval 100 \
     --smt z3 \
+    --log \
+    /home/sguest/work/haskell-backend/evm-semantics/tests/specs/erc20/ds/totalSupply-spec.debug-log \
```

Comment:
Ah need to `sed` the script as well, will address this

Reply:
Why are we even producing these logs? This should probably be removed from the `evm-semantics` code.
But yes, a `sed` script to kill the `--log` and its argument (either one or two lines, depending on `\` being present)  is good enough. Or I'd suggest to rather  just fix this manually now, we are not running this  script automatically and the correct fix is to change `evm-semantics`.